common/main/cpp/native_c4listener.cc

```diff
@@ -749,17 +754,23 @@ Java_com_couchbase_lite_internal_core_impl_NativeC4KeyPair_generateSelfSignedCer
     auto keys = (C4KeyPair *) c4KeyPair;
 
     int size = env->GetArrayLength(nameComponents);
-    auto subjectName = new C4CertNameComponent[size];
+    C4CertNameComponent *subjectName = new C4CertNameComponent[size];
```

Comment:
I see what you are doing with the rejection of auto but it was kind of made for situations like this where the object name shows up twice in one line (comment only, not a review)

Reply:
Yeah.... Overzealous.  WIll use "auto" with "new" in the future.